src/main/java/views/blackjack/blackjack.ftl.html

```diff
@@ -123,6 +123,41 @@
      });
 
 });
+
+//Function to handle hit requests from the user
+//Checks if the field of the hit button is "Hit"
+//If true then we make a post request to get a new card
+//Grab this new card, string configure it, and append
+//the new card to the users hand
+$("#dealHit").click(function(){
+    val = "";
+    data = "";
+    data = $("div.dealHit").text;
+
+    if(data == "Hit"){
+        var newCard;
+        //Send game data to get card{amount}
+        $.ajax({
+            type: "POST",
+            url: "/val/" + val,
+            data: JSON.stringify(game),
+            success: function(data, status){
+            console.log("Data: " + data + "\nStatus: " + status);
+            //Display game data
+            display(data);
+            game = data;},
+            contentType:"application/json; charset=utf-8",
+            dataType:"json",
+        });
```

Comment:
For the code after the post it looks like you are setting variables and displaying. The problem with ajax calls is that they are asynchronous. Some browsers wait for it to finish and some don't so it is possible that this code will execute before the update from the model finishes which will cause some display errors. Also ajax calls can also fail and not update the game which will cause additional errors. This code should really be in the success function for the ajax call